website/docs/concepts/provider_types.mdx

```diff
@@ -0,0 +1,178 @@
+## Different Types of Providers
```

Comment:
Is this still necessary? Now that the website has "API reference > all providers"

Reply:
Ahh, I didn't see that section get added. Is that section referenced throughout the rest of the documentation? I feel like it belongs in the main concepts portion of the documentation because there are so many questions about when and how to use different providers.